client/js/widgets/scoreboard.js

```diff
@@ -78,13 +79,15 @@ class Scoreboard extends Widget {
               type: 'select',
               label: 'Player',
               options: players,
-              variable: 'player'
+              variable: 'player',
+              value: players.find(player => player.text == playerName).value
```

Comment:
If `playerName` is not found, then `players.find()` returns `undefined`. Trying to access the `value` property of `undefined` is an error.

Reply:
I tried something like
`players.find(player => player.text == playerName)?.value ?? null`.  I changed null to ' ', players[0].value, and several other things trying to fix the team problem, but none of that worked.

Reply:
It looks like `players.find(player => player.text == playerName)?.value` should return `undefined` when `playersName` is not found (equivalent of what my code does. Not sure what `showInputOverlay` does with `null` or `''`, but `undefined` should behave as if `value` was not set at all.